tdrs-frontend/docker-compose.local.yml

```diff
@@ -1,10 +1,12 @@
-# Local version of Docker compose, using the Dockerfile.local instead and mounting node_modules
-version: "3"
+# Local version of Docker compose, using the localdev docker build target
+# instead and additionally mounts node_modules for faster builds
+version: "3.4"
```

Comment:
The `target` option was added in this version so we needed to upgrade in order to be able to specify a specific stage in a multi-stage dockerfile.